themes/10up-theme/includes/tailwind-helpers.js

```diff
@@ -0,0 +1,58 @@
+const themeJSON = require('../theme.json');
```

Comment:
Overall here, I like this feature. Just curious of a few things.

1. Could we add a header comment to this file, explaining its purpose?
2. Could we add JSDoc commenting to each of the functions?
3. Would this ever be problematic functionality if the theme.json had certain things removed by an engineer?

Reply:
@devinle - the above points sound good - @iansvo got any bandwidth to tackle this?

Reply:
Didn't see this until now but yeah I'll see about addressing a few things like this. 

@devinle Can you elaborate a bit more on number 3 for me?